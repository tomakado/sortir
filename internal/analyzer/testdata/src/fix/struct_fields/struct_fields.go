package struct_fields

type UnsortedStruct struct {
	Zebra string
	Apple int // want "struct fields are not sorted"
	Banana bool `json:"banana"`
}

type AnotherStruct struct {
	Z int
	A int // want "struct fields are not sorted"
	B int
}