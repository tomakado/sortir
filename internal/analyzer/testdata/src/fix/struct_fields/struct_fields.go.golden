package struct_fields

type UnsortedStruct struct {
	Apple int
	Banana bool `json:"banana"`
	Zebra string
}

type AnotherStruct struct {
	A int
	B int
	Z int
}